inc/relationship/admin-columns.php

```diff
@@ -233,7 +233,9 @@ private function get_column_data( $object_id, $object_type, $direction ) {
 	}
 
 	private function get_post_items( $object_id, $direction ) {
+		$post_types = get_post_types( '', 'names' );
 		$query = new WP_Query( [
+			'post_type'           => $post_types,
```

Comment:
Chỗ này giống trick nhỉ :D. Đúng ra là phải truyền `post_type` của chính relationship đó. Truyền kiểu này tuy cũng chạy trong phần lớn trường hợp, nhưng hơi ko chuẩn lắm :D.

Em thử improve code, tìm cách lấy ra đúng `post_type` mà relationship đăng ký xem sao.

Reply:
- Chỗ này mình có `$direction` rồi, nên em lấy cụ thể luôn từ đó, ko cần phải lấy cả `from` và `to` nữa.
- Và đổi tên biến `$mbr` thành `$relationship` nhé.
- Fix luôn coding standards.